### DISCUSSION TITLE
proxy support for Directus 8.6.2

### DISCUSSION THREAD

**Discussion by sbonasu** [directus/api#2536] [Q&A] [answered]
Hi,

  I have directus V8 php running behind apache on ubuntu. SSO to okta is configured and working as expected. Our security has asked to move server to a zone where outbound traffic goes through corporate proxy. When I deployed the app in new zone, SSO stopped working. I see Okta url timing out in the logs. Is there a way to proxy php traffic through proxy endpoints http_proxy and https_proxy?
  I'm able to manually curl to okta url from new host so I'm thinking proxy config is missing in directus
  Error
  ```
  error.2022-11-30.log:[2022-11-30 20:24:22] api[directus].ERROR: GuzzleHttp\Exception\ConnectException: cURL error 28: Failed to connect to xxxxxx.okta.com port 443: Connection timed out (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)
  ```

**Comment by sbonasu** [comment, marked as answer]
added env vars to apache

**Comment by rijkvanzanten** [comment]
v8 has been deprecated and has been end-of-life for over 2 years. Please don't rely on it in production. Use [the latest version](https://github.com/directus/directus/releases) instead.
